core/kazoo_services/src/kz_services.erl

```diff
@@ -683,30 +691,30 @@ is_good_standing_fold(Services, _Options, []) ->
     lager:debug("account ~s ran out of good funs, the ugly"
                ,[account_id(Services)]
                ),
-    'false';
+    {'false', <<"has not enough credit or no payment token">>};
```

Comment:
@icehess the "has" in these errors is odd.  It makes it so they are not stand alone:

{'error', "has not enough credit or no payment token"} vs {'error', "not enough credit or no payment token"}

I suspect you are doing that because logging/API errors but why not just make the 'has' part of that format?

(PS> applies to the other reasons like "has exceeded the maximum postpay amount", additionally in that case they "will" exceeded if allowed to debit the proposed amount...)